scylla-cql/src/frame/value.rs

```diff
@@ -364,6 +364,22 @@ impl std::hash::Hash for CqlVarint {
     }
 }
 
+/// Native CQL `decimal` representation.
+///
+/// Represented as a pair:
+/// - a [`CqlVarint`] value
+/// - 64-bit integer which determines the position of the decimal point
+///
+/// The type is not very useful in most use cases.
+/// However, users can make use of more complex types
+/// such as `bigdecimal::BigDecimal` (v0.2/v0.3/v0.4).
+/// The library support (e.g. conversion from [`CqlValue`]) for these types is
+/// enabled via `bigdecimal-02`, `bigdecimal-03` and `bigdecimal-04` crate features.
+///
+/// # DB data format
+/// Notice that [constructors](CqlDecimal#impl-CqlDecimal)
+/// don't perform any normalization on the provided data.
+/// For more details, see [`CqlVarint`] documentation.
```

Comment:
Not sure if I'm a fan of adding docstrings in a later commit. I think they should be introduced in the same commit as the item they document so that they provide more context for the reviewer when they go through the PR commit by commit.